subsys/bootloader/Kconfig

```diff
@@ -31,6 +49,27 @@ config SB_SIGNING_KEY_FILE
 	  '-DSB_SIGNING_KEY_FILE=/path/to/my/pem' when running cmake.
 	  See also SB_PUBLIC_KEY_FILES.
 
+config SB_SIGNING_COMMAND
+	string
+	prompt "Custom signing command" if !SB_PRIVATE_KEY_HANDLING
+	default ""
+	help
+	  This command will be called to produce a signature of the firmware.
+	  It will be called as "${CONFIG_SB_SIGNING_COMMAND} <file>"
+	  The command must take calculate the signature over the the contents
+	  of the <file> and write the signature to stdout.
+	  The signature must be on DER format.
+
+config SB_SIGNING_PUBLIC_KEY
```

Comment:
Should this be hidden when `SB_SIGNING_CUSTOM` is not selected?

Reply:
Yes, it is (see prompt).

Reply:
Have you tested it? My question was based on running `menuconfig` not reading code.